### ISSUE TITLE
Improvement: Respect .forceIgnore

### ISSUE THREAD

**Issue by ralphcallaway** [ChuckJonas/force-source-clean#2]
if someone wanted to store something else in a force.com package folder (i.e. selenium tests) that isn't subject to the compile on save, pull/push commands, seems like it would be expected this command would respect `.forceIgnore`

**Comment by ChuckJonas** [comment]
I haven't used `.forceIngore` to fully understand the sytnax... Here's [the docs](https://developer.salesforce.com/docs/atlas.en-us.sfdx_dev.meta/sfdx_dev/sfdx_dev_exclude_source.htm).

It's not immediately obvious to me if it's just using glob patterns or actually taking a more metadata context aware approach.

If it is just using globs, [npm ignore](https://www.npmjs.com/package/ignore) makes it super easy to check

**Comment by ralphcallaway** [comment]
oh good question, i'd assumed it was globs, but can't say for sure. my reading of the docs doesn't seem to conflict with it being implemented with globs, i guess you could crack open the dependencies for the four commands that utilize .forceIgnore if this came up

but honestly this feels like a feature that wouldn't matter until there is fairly widespread usage, guessing there aren't a ton of people putting non-sfdx files in a sfdx folder

**Comment by ralphcallaway** [comment]
i.e. sample syntax is all "globby"
```
# Specify a relative path to a directory from the project root
helloWorld/main/default/classes

# Specify a wildcard directory - any directory named “classes” is excluded
**classes

# Specify file extensions
**.cls
**.pdf

# Specify a specific file
helloWorld/main/default/HelloWorld.cls
```

**Comment by ChuckJonas** [comment]
Made a pass... https://github.com/ChuckJonas/force-source-clean/commit/9225647d94bb034578e28807d8c8e912148e9f25


Needs more testing

**Comment by ChuckJonas** [comment]
included in 0.0.4

Pretty limited testing

**Comment by ralphcallaway** [comment]
seems to be working well, had a clean failing due to merge conflict artifacts, added `**.orig` to `.forceIgnore` and was able to run without having to track down all the artifacts
